src/pages/SelectInterests.js

```diff
@@ -0,0 +1,58 @@
+import { isSupportedBrowser } from "../utils/misc/browserDetection";
+import useSelectInterest from "../hooks/useSelectInterest";
+import InfoPage from "./info_page_shared/InfoPage";
+import Header from "./info_page_shared/Header";
+import Heading from "./info_page_shared/Heading";
+import Main from "./info_page_shared/Main";
+import ButtonContainer from "./info_page_shared/ButtonContainer";
+import Footer from "./info_page_shared/Footer";
+import Button from "./info_page_shared/Button";
+import Tag from "./info_page_shared/Tag";
+import TagContainer from "./info_page_shared/TagContainer";
+import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
+
+export default function SelectInterests({ api, hasExtension }) {
+  const { allTopics, subscribedTopics, toggleTopicSubscription } =
+    useSelectInterest(api);
+
+  function navigateToNextPage() {
+    if (isSupportedBrowser() && hasExtension === false) {
+      return "/install_extension";
+    } else return "/articles";
+  }
+
+  return (
+    <InfoPage>
+      <Header>
+        <Heading>What would you like to read about?</Heading>
+      </Header>
+      <Main>
+        <TagContainer>
+          {allTopics.map((topic) => (
+            <Tag
+              key={topic.id}
+              className={
```

Comment:
wouldn't it make sense to also export an `isSubscribed(topic) {...}` function also from the reusable hook? It would look nice there with all the other functions that are handling subscriptions and topics I think!

Reply:
✅ Done